mantidproject/mantid
# Update to v6.3 release notes

**Description of work.**

To reduce merge conflicts release notes added during release have been added as separate files. This PR collates all these notes into the main V6.3 release notes and undertakes cleaning and formatting to improve quality and consistency.

In this PR the following release notes have been updated:-

- Diffraction
- Direct Geometry
- Framework
- Indirect Geometry
- Muon
- SANS
- Workbench

**To test:**

- remove `build/docs/doctrees`
- build the documents again and check there are no warnings
- Check that the links work
- Check images display as expected
- Check the information is still correct
- Grammar/Spelling check


*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
